filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -1286,7 +1286,12 @@ bool OpenGLDriver::isTextureFormatSupported(TextureFormat format) {
         return gl.ext.texture_compression_etc2;
     }
     if (isS3TCCompression(format)) {
-        return gl.ext.texture_compression_s3tc;
+        if (isS3TCSRGBCompression(format))		{
```

Comment:
Formatting issues: extra spaces on this line, `else {` should be on the same line as the closing `}` from the if

Reply:
Thanks.  I've pushed up this change.